core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java

```diff
@@ -0,0 +1,178 @@
+package greencity.webcontroller;
+
+import static greencity.dto.genericresponse.GenericResponseDto.buildGenericResponseDto;
+
+import greencity.constant.HttpStatuses;
+import greencity.dto.PageableDto;
+import greencity.dto.genericresponse.GenericResponseDto;
+import greencity.dto.habitfact.*;
+import greencity.service.HabitFactService;
+import greencity.service.HabitFactTranslationService;
+import greencity.service.LanguageService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import springfox.documentation.annotations.ApiIgnore;
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
```

Comment:
Change to RestCOntroller annotation - and won't be no need in adding @ResponseBody to methods